Sources/Plasma/FeatureLib/pfPython/cyAvatar.cpp

```diff
@@ -1620,6 +1620,27 @@ bool cyAvatar::LoadClothingFromFile(plFileName filename)
     return false;
 }
 
+PyObject* cyAvatar::FindBone(const ST::string& boneName)
+{
+    // make sure that there is atleast one avatar scene object attached (should be)
+    if (!fRecvr.empty() && fRecvr[0] != nullptr)
+    {
+        // find the armature modifier
+        plArmatureMod* avatar = (plArmatureMod*)IFindArmatureMod((plKey)fRecvr[0]);
```

Comment:
Is either of these casts necessary? A `const plArmatureMod*` should be enough (`plArmatureMod::FindBone` is `const`) and `fRecvr` is a vector of `plKeys`.

```suggestion
        const plArmatureMod* avatar = IFindArmatureMod(fRecvr[0]);
```

Reply:
This seems like it should work (and compiles fine), but the original pattern is used in pretty much every function in this file, so maybe we'll want to go through and to a cleanup task at some point.